src/equations/compressible_euler_quasi_1d.jl

```diff
@@ -0,0 +1,371 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    CompressibleEulerEquationsQuasi1D(gamma)
+
+The quasi-1d compressible Euler equations
```

Comment:
Could you elaborate (or ideally point to a reference) that explains the notion of quasi one-dimensionality?
Maybe reference the paper from which you take the numerical flux already here.